### PR TITLE
Translate `what's new for m59` into Simplified Chinese, resolve #4420

### LINKED ISSUE #4420: Translate `what's new for m59` into Simplified Chinese

### PR DESCRIPTION
**Pull request by wayou**
this PR also contains changes of pending PR #4331, #4384

### CODE REVIEW

File: src/content/zh-cn/updates/2017/04/devtools-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+# DevTools 更新点 (Chrome 59) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+欢迎来到新一期的 DevTools 更新日志。以下是 Chrome 59 中的新功能。
+
+提示：你可以访问 `chorme://help` 来查看当前使用的 Chrome 版本。
```
Comment (petele) [inline src/content/zh-cn/updates/2017/04/devtools-release-notes.md]: this should be `chrome://help` not `chOrme`

**Comment by agektmr** [comment]
@limhenry @kennith can either of you please help us review this?
@wayou Sorry for keeping you wait. We'll merge this as soon as you get LGTM.

**Comment by limhenry** [comment]
@agektmr Looks good for me 😄

**Comment by limhenry** [comment]
wait. this article is not synced with commit #4645

**Pull request merged**